vhost-device-gpu/README.md

```diff
@@ -56,6 +56,12 @@ A virtio-gpu device using the vhost-user protocol.
           [default: true]
           [possible values: true, false]
 
+      --gpu-path <PATH>
+          GPU device path (e.g., /dev/dri/renderD128)
+
+          [Optional] Specifies which GPU device to use for rendering. Only
+          applicable when using the virglrenderer backend.
```

Comment:
What about `--gpu-device`?, "-path" seems redundant. Could be  `--gpu-device-file`, but I think is too verbose